deps/CMakeLists.txt

```diff
@@ -17,9 +13,6 @@ function(mapbox_base_add_library name include_path)
     target_link_libraries(mapbox-base INTERFACE mapbox-base-${name})
 endfunction()
 
-add_library(mapbox-base INTERFACE)
```

Comment:
Please don't remove the `mapbox-base` target, we check if this target exists to prevent duplicated target entries when cascading dependencies attempt to include `mapbox-base` multiple times.

Reply:
https://github.com/mapbox/mapbox-base/blob/kdn-dev/include/CMakeLists.txt